README.md

```diff
@@ -1,30 +1,34 @@
 # arcgis-js-api
 
-A minified, unbuilt version of the [ArcGIS API for JavaScript](https://developers.arcgis.com/javascript/) to be used with [bower](http://bower.io).
+A minified, unbuilt version of the [ArcGIS API for JavaScript](https://developers.arcgis.com/javascript/) to be used with bower and npm.
```

Comment:
Could we keep the bower link even if we add npm?

Reply:
in my first draft i had links for both and looking at the markdown preview it just seemed distracting (especially given that the same links are repeated immediately below in `Features`).

that said, i'm happy to add it back (and make any other tweaks you like).

Reply:
done.